javascript/ecmascript-2015/array-updates/searching-array-methods.md

```diff
@@ -133,15 +116,15 @@ ints.???(???)
 
 Fill the array with the first number divisible by `2` and `3`:
 
-```
-let ints = []'2','5','4','5','6','5','8'];
-ints.???(ints.???(???))
+```javacript
+let ints = ['2','5','4','5','6','5','8'];
+ints.???(function (e) {???})
```

Comment:
Ok so the original question:

```js
let ints = []'2','5','4','5','6','5','8'];
ints.???(ints.???(???))
```

had the solution as

```js
let ints = []'2','5','4','5','6','5','8'];
ints.fill(ints.find(e => e % 4 === 0))
```

After you made your changes, the original question

> Fill the array with the first number divisible by `2` and `3`:

Doesn't make any sense.